homeassistant/components/sensor/integration.py

```diff
@@ -26,6 +26,12 @@
 CONF_UNIT_PREFIX = 'unit_prefix'
 CONF_UNIT_TIME = 'unit_time'
 CONF_UNIT_OF_MEASUREMENT = 'unit'
+CONF_INTEGRATION_METHOD = 'method'
```

Comment:
I think the `CONF_` key should match the value. Makes reading the code easier.

Reply:
👍